packages/block-editor/src/components/block-navigation/block.js

```diff
@@ -22,6 +22,7 @@ import DescenderLines from './descender-lines';
 import BlockNavigationBlockContents from './block-contents';
 import BlockSettingsDropdown from '../block-settings-menu/block-settings-dropdown';
 import { useBlockNavigationContext } from './context';
+import EllipsisMenuControls from './ellipsis-menu-controls';
```

Comment:
I wonder is if this name is OK 😁 `EllipsisMenuControls` should maybe become `BlockSettingsMenuControls` and this `BlockSettingsMenu` name to be propagated. `EllipsisMenu` sounds like the menu of some ellipsis and the real name of the menu is Block Settings.

Reply:
Agreed, the name ellipsis menu is... not fantastic. On the other hand the menu we want to implement here should be a superset of `BlockSettingsMenu`. See the comment here: https://github.com/WordPress/gutenberg/pull/22427#discussion_r428552883